### DISCUSSION TITLE
CMAKE_CUDA_STANDARD 17

### DISCUSSION THREAD

**Discussion by yrodriguez31** [OE4T/meta-tegra#1526] [Q&A]
Hello,
I working on the transition for L4T toyocto with  meta-tegra on branch kirkstone l4tR32.7.4.
I finaly have a full integration of my apps but i have  a regression on CMAKE_CUDA_STANDARD from 17 to 14.
DO you have an idea how to use CMAKE_CUDA_STANDARD 17 with meta-tegra ?

Thaks 
Yannick

**Comment by madisongh** [comment]
The version of CUDA that NVIDIA supplied for L4T R32.7.x/JetPack 4 is 10.2, which [evidently does not support c++17](https://docs.nvidia.com/cuda/archive/10.2/cuda-compiler-driver-nvcc/index.html#options-for-altering-compiler-linker-behavior-std) for the `--std` flag passed to `nvcc`.

**Reply by yrodriguez31** [reply]
Ok thanks for your quick reply.
